ui/src/views/Quickstart/index.jsx

```diff
@@ -359,19 +359,24 @@ export default class QuickStart extends Component {
                 For independent developers and organization owners: How to set
                 up your repository with {window.env.APPLICATION_NAME}
               </Typography>
-              <Typography paragraph>
-                <ul>
-                  <li>
```

Comment:
So, as it stands, React renders the following lines to HTML, in a way something like this:

```<p>
        <ul> </ul>
   </p>
```
According to this [document](https://www.w3.org/TR/html401/struct/text.html#h-9.3.1), a ```<p></p>``` tag can only contain inline elements. That means putting a <div></div> tag inside it should be improper since the div tag is a block element. Improper nesting might cause glitches like rendering extra tags, which can affect your javascript and CSS.